frontend/src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/WorkItems/View.tsx

```diff
@@ -101,7 +115,8 @@ export default function View({
           {some(contributions, { ignored: true }) && (
             <div className="flex flex-row items-center gap-2 font-walsheim text-sm font-normal text-greyscale-50">
               <EyeOffLine />
-              {T("reward.form.contributions.showIgnored")}
+
+              <div className="sm:inline lg:hidden xl:flex">{T("reward.form.contributions.showIgnored")}</div>
```

Comment:
Between mobile and the `sm` breakpoint it would be block before going inline.
```suggestion
              <div className="inline lg:hidden xl:flex">{T("reward.form.contributions.showIgnored")}</div>
```